release.sh

```diff
@@ -23,35 +31,57 @@ if [[ ! $VERSION =~ ^[0-9]*\.[0-9]*\.[0-9]*(-[A-Z0-9]*)?$ ]]; then
 fi
 
 GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
-if [[ ! $GIT_BRANCH =~ main ]]; then
+if [[ ! $GIT_BRANCH =~ $UPSTREAM_BRANCH ]]; then
     echo "You do not seem to be on the main branch!"
```

Comment:
maybe also use the variable here? echo "You do not seem to be on the ${UPSTREAM_BRANCH} branch!"